src/content/app/genome-browser/state/browser-bookmarks/browserBookmarksSlice.ts

```diff
@@ -46,7 +45,7 @@ export type PreviouslyViewedObjects = {
 };
 
 export type BrowserBookmarksStateForGenome = {
-  bookmarks: [];
+  bookmarks: PreviouslyViewedObject[];
```

Comment:
Well, technically, we don't know anything about the shape of bookmarks (the original intention was that they would be the bookmarks that the user creates manually). We aren't using this array at all in our code; we are just keeping this as a placeholder and a reminder of what's likely to come. So I would advise against this change.

Reply:
✅